.github/scripts/ci.sh

```diff
@@ -5,19 +5,41 @@ set -eu -o pipefail
 
 DIR="${PWD}"
 
-curl --silent --show-error --location https://raw.githubusercontent.com/hyperledger/fabric/main/scripts/install-fabric.sh | bash -s -- --fabric-version "$1"
+# for fabric 2.5 we don't test for BFT
+if [[ $1 == '2.5.3' && $3 == 'BFT' ]]; then
+    exit 0
+fi
```

Comment:
Perhaps this test could be done in golang.yaml instead of here so this script is never driven with invalid parameter combinations?